src/group_theory/subgroup.lean

```diff
@@ -560,6 +560,10 @@ lemma closure_union (s t : set G) : closure (s ∪ t) = closure s ⊔ closure t
 lemma closure_Union {ι} (s : ι → set G) : closure (⋃ i, s i) = ⨆ i, closure (s i) :=
 (subgroup.gi G).gc.l_supr
 
+lemma closure_eq_bot_iff (G : Type*) [group G] (S : set G) :
```

Comment:
Does `to_additive` work on this?

Reply:
Oh silly me! Thanks! It does. Well spotted.